doc/source/tutorials/bipartite_matching/bipartite_matching.rst

```diff
@@ -0,0 +1,60 @@
+==========================
+Maximal Bipartite Matching
```

Comment:
"Maximum", not "maximal". It's not the same.

If we discuss maximum bipartite matchings, it's important to point out that there is already a function for this. This is a nice example, but people should not get the impression that they need to resort to flow calculations to get a bipartite matching. It would be nice to compare with the result of `.maximum_bipartite_matching()`.

Reply:
My bad, I missed that particular method while going through the API. I've modified the example to use `.maximum_bipartite_matching()` and moved the original tutorial to the `archived/` folder. 

I would like these examples to show the most efficient / standard way of doing things, so I don't think that calculating the matching by manually making a max flow graph belongs here.

Reply:
I do think it was a very good and educational example, and I would very much suggest keeping it. Just make it clear that this is a not the most efficient way of finding a maximum bipartite matching.

As I see it, this example is not about a specific task (bipartite matching). Instead, it demonstrates how to work with igraph in general, and gives you the opportunity to show several basic operations, such as adding edges, setting up a bipartite graph, retrieving results from a flow object, highlighting a subset of edges, etc. For this, it's an excellent example.

Reply:
In fact, I would encourage you to try to show off even more functionality within the context of this example, such as `is_biparite`, `Graph.Bipartite`, or whatever you think fits. It's nice to show more than one way of doing things, such as setting the `type` property directly or through `Graph.Bipartite`.

Reply:
That makes complete sense, and I'll let my supervisor @iosonofabio make the final call. He's currently on break, so I'll leave this in the archives for now.

Reply:
Happy to go with whatever @szhorvat proposed, even if it makes it a little longer